examples/fabric-bridge-app/linux/RpcServer.cpp

```diff
@@ -64,6 +65,21 @@ pw::Status FabricBridge::AddSynchronizedDevice(const chip_rpc_SynchronizedDevice
     return pw::OkStatus();
 }
 
+pw::Status FabricBridge::RemoveSynchronizedDevice(const chip_rpc_SynchronizedDevice & request, pw_protobuf_Empty & response)
+{
+    NodeId nodeId = request.node_id;
+    ChipLogProgress(NotSpecified, "Received RemoveSynchronizedDevice: " ChipLogFormatX64, ChipLogValueX64(nodeId));
+
+    int result = DeviceMgr().RemoveDeviceByNodeId(nodeId);
+    if (result == -1)
```

Comment:
```suggestion
    int removed_idx = DeviceMgr().RemoveDeviceByNodeId(nodeId);
    if (removed_idx < 0)
```

just "result" is not as clear about it's meaning and why -1 is special.